src/ui/components/Video/RecordedCursor.tsx

```diff
@@ -2,12 +2,16 @@ import assert from "assert";
 import { useLayoutEffect, useRef } from "react";
 
 import { findMostRecentClickEvent, findMostRecentMouseEvent } from "protocol/RecordedEventsCache";
+import { useCurrentFocusWindow } from "replay-next/src/hooks/useCurrentFocusWindow";
 import { state } from "ui/components/Video/imperative/MutableGraphicsState";
 
-const CLICK_TIMING_THRESHOLD_MS = 200;
+// One frame seems reasonable for a click to be considered
+// recent enough to be drawn on the screen
+const CLICK_TIMING_THRESHOLD_MS = 16.67;
 
 export function RecordedCursor() {
   const elementRef = useRef<HTMLDivElement>(null);
+  const focusWindow = useCurrentFocusWindow();
```

Comment:
We aren't using `focusWindow` anymore so we can remove this hook and the dependency below
```suggestion
```